src/functions.js

```diff
@@ -516,15 +517,42 @@ defineFunction([
     };
 });
 
-// Left and right overlap functions
-defineFunction(["\\llap", "\\rlap"], {
+// Horizontal overlap functions
+defineFunction(["\\mathllap", "\\mathrlap", "\\mathclap"], {
     numArgs: 1,
     allowedInText: true,
 }, function(context, args) {
     const body = args[0];
     return {
-        type: context.funcName.slice(1),
+        type: "lap",
+        className: context.funcName.slice(5),
```

Comment:
I'd like to avoid including CSS specifics at this level, maybe this could be called `alignment`.